### ISSUE TITLE
Remove type field from subjects table

### ISSUE THREAD

**Issue by SiddheshRane** [OpenKatta/UniTimetable#9]
The subjects table has a type field which denotes whether a subject is theory or lab. Although this distinction feels natural to us, but for our application it is unnecessary. Our software does not handle any constraints for lab or theory. It will not flag a warning if a Chemistry lab is alloted venue in FOSS lab or seminar hall. It does not use the field in any meaningful way besides complicating existing code with certain hacks.

In summary we need a unified concept of a subject and leave special types handling to the user or to another table in future
